docs/CONTRIBUTORS.rst

```diff
@@ -125,6 +125,26 @@ increase log level up to DEBUG ('-vvvv').
     $ python test_updater.py -vvvv TestUpdater.test_4_refresh # run test with DEBUG log level
 
 
+All of the log levels and the corresponding options that could be used for testing are:
+
+.. list-table::
+   :widths: 20 25
+   :header-rows: 1
+
+   * - Option
+     - Log Level
+   * - default (no argument passed)
+     - ERROR
+   * - `-v`
+     - ERROR + unittest module prints test names
```

Comment:
we should clarify that unit test names are printed when 1 or more `-v`'s are passed. As currently written it could be interpreted as only printed when one `-v` is passed. Perhaps something like:

```suggestion
     - ERROR (test names are not printed)
   * - `-v`
     - ERROR (test names are printed at this level and above)
```

Reply:
Used your suggestions and amended.